src/objecttypes/core/admin.py

```diff
@@ -139,3 +151,25 @@ def response_change(self, request, obj):
             return self.add_new_version(request, obj)
 
         return super().response_change(request, obj)
+
+    def import_github_url(self, request):
+        if request.method == "POST":
+            url = request.POST.get("github_url")
+            content = requests.get(url).json()
+
+            object_type = ObjectType.objects.create(
+                name=content["title"].title(),
+                name_plural=pluralize(content["title"]).title(),
```

Comment:
I don't think that writing and supporting our own noun pluralizer is a good idea.
The title can be in English (or other language), it can be some abbreviation, it can have zero meaning, etc. 

I think that only client should decide which plural name should have an object type. I suggest removing automatic pluralizer and adding two more fields in the importing form:
* `name`
* `name_plural`